tests/test_forms.py

```diff
@@ -0,0 +1,64 @@
+from unittest.mock import Mock
+
+from django.core import mail
+from django.test import TestCase
+
+from wagtail_extensions.forms import ContactForm
+
+
+class ContactFormTestCase(TestCase):
```

Comment:
The rest of this project's tests are written as py.test tests, I think it would be worth converting those and using the opportunity to rewrite the tests at the same time.